src/main/kotlin/skytils/skytilsmod/features/impl/handlers/AuctionData.kt

```diff
@@ -81,6 +86,13 @@ class AuctionData {
                 }
             }
         }
+        fixedRateTimer(name = "Skytils-FetchBazaarData", period = 10 * 60 * 1000L) {
+            if(Skytils.config.fetchLowestBINPrices && Skytils.config.containerSellValue) {
+                Skytils.hylinAPI.getBazaarData().whenComplete {
```

Comment:
idk if this is gonna break hypixel but hopefully not? i can just put all the bz prices on the lowestbin cause that's what it already does for certain whitelisted items

Reply:
Ideally we should try to reduce our api queries when possible